WidgetExtension/Common/PCWidgetColorScheme.swift

```diff
@@ -0,0 +1,122 @@
+import SwiftUI
+
+let widgetRedDark = Color(red: 217, green: 32, blue: 28)
```

Comment:
Instead of creating global constants you can do the following:

```
Extension Color {
  static let widgetRedDark = Color(red: 217, green: 32, blue: 28)
}
```
And then in any place you need it you just do: `.widgetRedDark`

Reply:
Done in [36762aa](https://github.com/Automattic/pocket-casts-ios/pull/1343/commits/36762aa353bdc94b9a933004dafaf6cff274899c)